core/frontend/src/components/health/GpsTrayMenu.vue

```diff
@@ -1,18 +1,17 @@
 <template>
-  <v-row :v-if="gps_connected" style="display: contents;">
-    <v-badge
-      :content="satellites_number"
-      :value="satellites_number"
-      :title="connection_description"
-      :color="number_color"
-      class="mr-2"
-      overlap
-    >
-      <v-icon color="white">
-        {{ satellites_number ? "mdi-crosshairs-gps" : "mdi-crosshairs-question" }}
-      </v-icon>
-    </v-badge>
-  </v-row>
+  <v-badge
+    v-if="gps_connected"
+    :content="satellites_number"
+    :value="satellites_number"
+    :title="`${connection_description} (Number of satellites: ${satellites_number})`"
+    :color="number_color"
+    class="mr-2"
+    overlap
+  >
+    <v-icon color="white">
+      {{ satellites_number ? "mdi-crosshairs-gps" : "mdi-crosshairs-question" }}
```

Comment:
This is still very unintuitive to me, I'm afraid.

It's not the icon itself that's the issue (I can see why crosshairs would indicate position, even if it's not super familiar to me), my concern is with the combination of crosshairs + number of satellites, because there aren't multiple locations being stored, or multiple GPS sensors being averaged between to estimate the location, which is how my brain wants to interpret that display.

Reply:
There is a tooltip saying that is the number of satellites, but I'm open to suggestions.

Reply:
maybe the type of fix? like "rtk", "dgps", "3d" and friends?

Reply:
This is already in the description.

Reply:
> There is a tooltip saying that is the number of satellites ...

That helps, but feels like the reasoning is backwards - ideally the reaction to an interface should be "it makes sense, and has extra detail if you're interested", rather than "it's confusing so we had to explain it". 

> ... I'm taking suggestions

1. I'd suggest using a satellite icon when there is a valid satellite lock
2. If that's not possible, I think just the word `GPS` would be easier to understand than the target icon (when there's a satellite number connected to it)
3. If the target icon is critical then I think the satellite number should _only_ be included in the tooltip, to avoid confusion, although that does then provide less information "at a glance" to people who know what they're looking at

Reply:
1. A satellite icon implies on satellite communication. (Latest iphone as example)
2. "GPS" does not follow our current UI where we use icons
3. I'm ok to remove the icon at all, but in general that'll result Ina worst UI in my opinion. Based on our current userbase I believe that the current UI is fine.

I also against removing the badge because we have different colors for different stats.

About the icon choice, for Android and google is pretty common.
![Screenshot_20240624-233340](https://github.com/bluerobotics/BlueOS/assets/1215497/3597e0db-052a-493c-add9-d44a30e93be7)
![Screenshot_20240624-233321](https://github.com/bluerobotics/BlueOS/assets/1215497/3d1f9929-9c89-40ef-83bb-dbba2addd74f)

Reply:
I think the issue with the icon is that this is icon is commonsly associated with an action "center on my current position", and is not usually representing a status. but yes, we can follow the discussion on an issue